packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -38,7 +39,7 @@
         <Icon name="{{ _iconName }}" isReusable={ false } />
     {% endif %}
     <span class="{{ _nameClassName }}">
-        <span class="text-truncate" data-spirit-populate-field="name">
+        <span class="{{ _textTruncate }}" data-spirit-populate-field="name">
             {{- _fileName -}}
         </span>
     </span>
```

Comment:
Oh my, could you please add classes in the if _onEdit section to the {# Class names #} section too?

Reply:
(it is below this code excerpt)

Reply:
Fixed, thanks for noticing :-)